frontend/src/screens/alby/AlbyAuthRedirect.tsx

```diff
@@ -11,7 +11,17 @@ export default function AlbyAuthRedirect() {
   const queryParams = new URLSearchParams(location.search);
   const forceLogin = !!queryParams.get("force_login");
   const url = info?.albyAuthUrl
-    ? `${info.albyAuthUrl}${forceLogin ? "&force_login=true" : ""}`
+    ? (() => {
+        const _url = new URL(info.albyAuthUrl);
+        if (forceLogin) {
+          _url.searchParams.append("force_login", "true");
+        }
+        if (info.albyUserIdentifier) {
+          _url.searchParams.append("identifier", info.albyUserIdentifier);
+        }
```

Comment:
this should be enough. if the user is logged in with that user then we don't need to prompt the login. so you can now emove the force login.?

Reply:
This is a different case. The force login is when the user explicitly wants to change the account they use for the hub

Reply:
ahh, true. 👍